quiz.css

```diff
@@ -0,0 +1,21 @@
+.cars{
+	background: lightblue;
+	min-height: 250px;
+	text-align: center;
+}
+
+.carClass{
+	background: goldenrod;
+	min-height: 250px;
+	text-align: center;
+}
+
+
+.clickedClass {
+	border: 4px solid blue;
+	background: grey;
+	text-align: center;
+	font-family: Helvetica;
+	min-height: 250px;
+}
```

Comment:
This class is no longer being used, so your borders aren't changing when a card is clicked